packages/runtime-core/src/componentRenderUtils.ts

```diff
@@ -37,7 +37,8 @@ export function markAttrsAccessed() {
 }
 
 export function renderComponentRoot(
-  instance: ComponentInternalInstance
+  instance: ComponentInternalInstance,
+  getFallthroughAttrs?: (attrs: Data) => Data | undefined
```

Comment:
There's another call site in `server-renderer/src/renderToString` that missed the update because the argument is declared as optional.

But in order to pass it to the server renderer we need to export `getFallthroughAttrs` from `@vue/runtime-dom`, which is not ideal (it gets exposed on the final global `Vue` as well).

I'm thinking we should probably keep the logic in core, since `class` and `style` have always had special treatment in core (e.g. the object and array syntax normalization). Most known custom renderers also supports `class` and `style` in a similar fashion to the DOM.

We can, however, still provide the option so that custom renderers can overwrite the default fallthrough with a simple `() => undefined`.